README.md

```diff
@@ -37,6 +37,14 @@ See the [Eth 2.0 Phase 2 Wiki](https://hackmd.io/UzysWse1Th240HELswKqVA?view) fo
 * [Beacon node API for validator](specs/validator/0_beacon-node-validator-api.md)
 
 
+## Additional specifications for client implementers
+
+Additional specifications and standards outside of core consensus can be found in the following repos:
```

Comment:
Hmm, networking spec is also "outside of core consensus" but we leave them here for more visibility/engagements, right? Could you elaborate why only moving restful APIs away? Is it for letting the clients take `eth2.0-apis` 
 as a submodule?

Reply:
When I said outside of core consensus, I meant more broadly, that a client doesn't really have to implement any of these APIs. This only aids in interoperability and standards for dev tooling. The cycle on which these might emerge and the standards around them is likely to be pretty independent of the core specs. I also think we might want to write tests against these APIs which would even more so clutter this repo. 

I mispoke with "core consensus". I meant more broadly, the things that a client _must_ conform to.